src/wallet/components/WalletDropdownFundLink.test.tsx

```diff
@@ -0,0 +1,97 @@
+import { fireEvent, render, screen } from '@testing-library/react';
+import { describe, expect, it, vi } from 'vitest';
+import { WalletDropdownFundLink } from './WalletDropdownFundLink';
+
+const FUNDING_URL =
+  'http://keys.coinbase.com/funding?dappName=&dappUrl=http://localhost:3000/';
+
+describe('WalletDropdownFundLink', () => {
+  it('renders correctly with default props', () => {
+    render(<WalletDropdownFundLink />);
+
+    const linkElement = screen.getByRole('link');
+    expect(linkElement).toBeInTheDocument();
+    expect(linkElement).toHaveAttribute('href', FUNDING_URL);
+    expect(screen.getByText('Fund wallet')).toBeInTheDocument();
```

Comment:
Github is showing some failure here. Are you seeing these?

Reply:
oops lol, forget i need to update the test every time i make a copy change here

Reply:
So that's the thing, if you write code that is more atomic, you can also write tests are less incline to change all the time.